vsphere-deployer/cluster.yaml.template

```diff
@@ -12,4 +12,7 @@ spec:
     providerConfig:
       value:
         apiVersion: "vsphereproviderconfig/v1alpha1"
-        kind: "VsphereProviderConfig"
\ No newline at end of file
+        kind: "VsphereClusterProviderConfig"
```

Comment:
The [README](https://github.com/karan/cluster-api/tree/73086ded2e4fbb514aeee4edc7dd238469dd63f9/vsphere-deployer) for vsphere-deployer will need to be updated to include instructions on creating the cluster.yaml file.

Reply:
Done